### PR TITLE
NL feedback client side support

### PR DESCRIPTION
**Pull request by shifucun**
image

### CODE REVIEW

File: server/routes/nl/api.py [hunk 1]
```diff
@@ -223,11 +223,11 @@ def feedback():
       not current_app.config['LOG_QUERY']):
     flask.abort(404)
 
-  session_info = request.json('session_info')
-  feedback_data = request.json['feedback_data']
+  session_id = request.json['sessionId']
+  feedback_data = request.json['feedbackData']
```
Comment (pradh) [inline server/routes/nl/api.py:227]: nit: Are you particular about camelCase?  Elsewhere the session-id uses snakecase (https://github.com/datacommonsorg/website/blob/b0331c234c77f303706cc5f9ac55faf197c4dae9/server/lib/nl/fulfillment/context.py#L139-L155)

**Review by pradh — COMMENTED** [review]
Nice!  Do you have a screenshot?

**Review by pradh — APPROVED** [review]

File: server/routes/nl/api.py (hunk 1 above)
Comment (shifucun) [reply server/routes/nl/api.py:227]: Json field should use camelCase..

**Comment by shifucun** [comment]
> Nice! Do you have a screenshot?

Added one. It looks a bit funny though..

**Pull request merged**